src/main/java/core/basesyntax/classes/figures/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.classes.figures;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.